Updating Documentation For Using LeRobot (#21)
TrossenRobotics/trossen_arm: docs/tutorials/lerobot/configuration.rst

========================================
Configuring Trossen AI Arms with LeRobot
========================================

In order to use the Trossen AI Arms with LeRobot, you need to first configure the arms with the necessary specifications.
The specifications include the IP address, model_name, and camera serial numbers.
The following steps will guide you on how to configure the Trossen AI Arms with LeRobot.

> 

Reviewer: ```suggestion
```

            "left": TrossenArmDriverConfig(
                ip="192.168.1.5",
                model="V0_FOLLOWER",
            ),
            "right": TrossenArmDriverConfig(
                ip="192.168.1.4",
                model = "V0_FOLLOWER",
            ),
        }
    )

    # Troubleshooting: If one of your IntelRealSense cameras freeze during
    # data recording due to bandwidth limit, you might need to plug the camera
    # on another USB hub or PCIe card.
    cameras: dict[str, CameraConfig] = field(
        default_factory=lambda: {
            "cam_high": IntelRealSenseCameraConfig(
                serial_number=130322270184,
                fps=30,
                width=640,
                height=480,
            ),
            "cam_left_wrist": IntelRealSenseCameraConfig(
                serial_number=218622274938,
                fps=30,
                width=640,
                height=480,
            ),
            "cam_right_wrist": IntelRealSenseCameraConfig(
                serial_number=128422271347,
                fps=30,
                width=640,
                height=480,
            ),
        }
    )

    mock: bool = False

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

Setup IP Address
----------------

> 
> To set up the IP address for the Trossen AI Arms, you must first ensure that the arms are connected to the same network as your computer.
> Refer to `Ethernet Setup <https://docs.trossenrobotics.com/trossen_arm/main/getting_started/software_setup.html#ethernet-setup>`_ for correct connection instructions.
> Once connected, you can find or configure the IP address using the `Configuration Demo <https://github.com/TrossenRobotics/trossen_arm/blob/main/demos/python/configure_cleanup.py>`_ or `Set IP Address Demo <https://github.com/TrossenRobotics/trossen_arm/blob/main/demos/python/set_ip_method.py>`_.
> 
> 

Reviewer: ```suggestion

To set up the IP address for the Trossen AI Arms, you must first ensure that the arms are connected to the same network as your computer.
Refer to :ref:`Ethernet Setup <getting_started/software_setup:Ethernet Setup>` for correct connection instructions.
Once connected, you can find or configure the IP address using the :ref:`getting_started/demo_scripts:`configure_cleanup`_` demo or the :ref:`getting_started/demo_scripts:`set_ip_method`_` and :ref:`getting_started/demo_scripts:`set_manual_ip`_` demos.

```

Camera Serial Number
--------------------

> 
> #. Open realsense-viewer
> 
>     .. code-block:: bash
> 
>         realsense-viewer
> 
>     .. note::
> 
>         If realsense-viewer is not already installed on your machine, follow `these steps on the librealsense GitHub repository <https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md>`_  to install ``librealsense2-utils``.
> 
> #. Plug in a single camera and check the sidebar for its entry. If it does not show up in the side bar, click Add Source and find the Intel RealSense D405 in the drop down.
> 
> #. Click on Info for the camera, find the Serial Number, and copy it.
> 
>     .. image:: images/rsviewer_serialno2.png
>         :alt: Realsense Viewer
>         :align: center
> 
> #. Put the camera serial number in the appropriate config entry at :guilabel:`lerobot/common/robot_devices/robots/configs.py`.
> 
> #. Repeat for the rest of the cameras. If the workspace has not been symbolically-linked, a rebuild may be necessary.
> 

Reviewer: ```suggestion

#.  Open realsense-viewer

    .. code-block:: bash

        realsense-viewer

    .. note::

        If realsense-viewer is not already installed on your machine, follow `these steps on the librealsense GitHub repository <https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md>`_  to install ``librealsense2-utils``.

#.  Plug in a single camera and check the sidebar for its entry.
    If it does not show up in the side bar, click Add Source and find the Intel RealSense D405 in the drop down.

#.  Click on Info for the camera, find the Serial Number, and copy it.

    .. image:: images/rsviewer_serialno2.png
        :alt: Realsense Viewer
        :align: center

#.  Put the camera serial number in the appropriate config entry at :guilabel:`lerobot/common/robot_devices/robots/configs.py`.

#.  Repeat for the rest of the cameras.

```

========================================
Configuring Trossen AI Arms with LeRobot
========================================

> Inorder to use the Trossen AI Arms with LeRobot, you need to configure the armsby providing the necessary specifications.

Reviewer: ```suggestion
In order to use the Trossen AI Arms with LeRobot, you need to first configure the arms with the necessary specifications.
```

> The specifications include the ip address,model_name, and camera serial number.

Reviewer: ```suggestion
The specifications include the IP address, model_name, and camera serial numbers.
```

            "right": TrossenArmDriverConfig(
                ip="192.168.1.4",
                model = "V0_FOLLOWER",
            ),
        }
    )

    # Troubleshooting: If one of your IntelRealSense cameras freeze during
    # data recording due to bandwidth limit, you might need to plug the camera
    # on another USB hub or PCIe card.
    cameras: dict[str, CameraConfig] = field(
        default_factory=lambda: {
            "cam_high": IntelRealSenseCameraConfig(
                serial_number=130322270184,
                fps=30,
                width=640,
                height=480,
            ),
            "cam_left_wrist": IntelRealSenseCameraConfig(
                serial_number=218622274938,
                fps=30,
                width=640,
                height=480,
            ),
            "cam_right_wrist": IntelRealSenseCameraConfig(
                serial_number=128422271347,
                fps=30,
                width=640,
                height=480,
            ),
        }
    )

    mock: bool = False

Setup IP Address
----------------

To setup the IP address for the Trossen AI Arms, you need to connect the arms to the same network as your computer.

> You can then find the IP address/ Set them using the following documentation.

Reviewer: ```suggestion
You can then find/set the IP address using the following documentation.
```

> You can then find the IP address/ Set them using the following documentation.

Reviewer: > using the following documentation

missing link
